testgrid/config/config.yaml

```diff
@@ -1806,12 +1806,16 @@ dashboards:
   dashboard_tab:
   - name: gke-cvm-1.6-cvm-master-upgrade-master
     test_group_name: ci-kubernetes-e2e-gke-cvm-stable-cvm-master-upgrade-master
+    description: 'Upgrade master only, in gke, from container_vm master-2 to container_vm master'
```

Comment:
In these cases I think `master_1` means the branch that we're upgrading to, and `master_2` means the thing that we're upgrading (just the master, not the cluster)

So I read this as "upgrade master only, in gke, from container_vm 1.6 to container_vm master"